test/run.mjs

```diff
@@ -48,7 +48,13 @@ if (nodeVersion !== expectedVersion) {
  */
 const replaceScript = (data) => {
   const localPath = path.join(__dirname, '..');
-  return data.replace(new RegExp(localPath, 'g'), '<local-path>');
+  return data
+    .replace(
+      // eslint-disable-next-line security/detect-non-literal-regexp -- Test code.
```

Comment:
I'm not happy about this either. Perhaps we should only enable security on non-test files?

Reply:
Depends what other rules `security` holds. I would like it help prevent if people try to do weird code executions for instance. But yeah, a regex like this is fine.